FSJ_django20_project/FSJ/views.py

```diff
@@ -46,11 +46,20 @@ def profile(request):
             profile_form = AdjudicatorRestrictedForm(instance=FSJ_user)
         elif isinstance(FSJ_user, Coordinator):
             profile_form = CoordinatorRestrictedForm(instance=FSJ_user)        
-            
+
     context = get_standard_context(FSJ_user)
     template = loader.get_template("FSJ/profile.html")
     context["form"] = profile_form
     url = "/profile/"
     context["url"] = url
     return HttpResponse(template.render(context, request))
 
+@login_required
+@user_passes_test(is_FSJ_user)
+def awards(request):
+    FSJ_user = get_FSJ_user(request.user.username)
+    if isinstance(FSJ_user, Coordinator):
+        return coordinator_awards(request, FSJ_user)
+    #elif isinstance(FSJ_user, Student):
```

Comment:
Remove commented out code if they're not necessary. Also, ensure you're handling what happens if the FSJ_User is NOT an instance of Coordinator since the test here is that any FSJUser may navigate to this link.

Reply:
I've removed the comment and changed this to only allow coordinators. When we implement a students award page, this will be changed